lib/db/queries.ts

```diff
@@ -138,7 +138,47 @@ export async function getMessageById({ id }: { id: string }) {
       .where(eq(message.id, id));
     return selectedMessage;
   } catch (error) {
-    console.error("Failed to get chat by id from database");
+    console.error("Failed to get message by id from database");
+    throw error;
+  }
+}
+
+export async function savePrediction({
+  content,
+  marketAddress,
+}: {
+  content: string;
+  marketAddress: string;
+}) {
+  try {
+    return await db
+      .insert(prediction)
+      .values({
+        createdAt: new Date(),
+        updatedAt: new Date(),
```

Comment:
timestamps should be added by default on DB side

Reply:
removing this generates a typescript error, we need to update the schema to use `.defaultNow()`, maybe that way will not complain about missing defaults.

Agree with you, but not sure if it's best to solve that on this PR.

Reply:
Created an [issue](https://linear.app/swaprhq/issue/PGIO-154/add-default-values-to-createdat-and-updatedat-fields)